metadata.json

```diff
@@ -90,10 +90,10 @@
   ],
   "description": "Manage SSH",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 6.0.0"},
-    {"name":"puppetlabs/concat","version_requirement":">= 2.0.0 < 6.0.0"},
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 7.0.0"},
+    {"name":"puppetlabs/concat","version_requirement":">= 2.0.0 < 7.0.0"},
     {"name":"ghoneycutt/common","version_requirement":">= 1.4.1 < 2.0.0"},
-    {"name":"puppetlabs/firewall","version_requirement":">= 1.9.0 < 2.0.0"},
-    {"name":"puppetlabs/sshkeys_core","version_requirement":">= 1.0.1 <2.0.0"}
+    {"name":"puppetlabs/firewall","version_requirement":">= 1.9.0 < 3.0.0"},
```

Comment:
what testing has been done to show that it actually works with these newer versions?

Reply:
We have been running the updated versions of the dependencies modules for a couple of months now on multiple production environments without any problems (except for the warnings initially indicated in this pull request when `puppet module list` command is run). We also have multiple modules of which we use the latest dependency modules and per puppet `pdk`, all unit tests runs fine and passes. In addition to this, we also manually updated the dependency versions in the metadata.json to the latest versions and as that resolved the warnings, no issues were encountered. If there are any additional specific tests you would like for me to perform, I will be more than happy to do so. Thanks for your help thus far.